src/components/FactSheetsData.js

```diff
@@ -0,0 +1,91 @@
+import { useState, useEffect } from "react";
+import styles from "@/styles/financial-reporting.module.scss";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { useTranslation } from "next-i18next";
+import { useSelector } from "react-redux";
+import { getFactsheetsState } from "@/store/factsheets/factsheets.slice";
+
+export const FactSheetsData = () => {
+  const { t } = useTranslation("common");
+  const router = useRouter();
+
+  const factsheetsData = useSelector(getFactsheetsState);
+
+  const [factSheetList, setFactSheetList] = useState([]);
+  const [activeFactSheetList, setActiveFactSheetList] = useState([]);
+  function factsheetTabHandler(e) {
```

Comment:
Add a space and do arrow function please :)

Reply:
done